libs/dh/market-participant/actors/feature-delegation/src/lib/create/dh-delegation-create-modal.component.ts

```diff
@@ -184,7 +184,12 @@ export class DhDelegationCreateModalComponent extends WattTypedModal<DhActorExte
           value: gridArea.id,
           displayValue: gridArea.displayName,
         }))
-      )
+      ),
+      tap((gridAreas) => {
+        this.createDelegationForm.patchValue({
+          gridAreas: gridAreas.map((gridArea) => gridArea.value),
```

Comment:
suggestion: add the same logic to the observable on line 171, or move it to a method :)